### PR TITLE
Include nyx_version flag in bookkeeper

### PR DESCRIPTION
**Pull request by jchavesmontero**
Include this flag for consistency with future versions of the Nyx file. Could you first run src/lace/scripts/compute_nyx_emu_cosmo.py and then Tutorial_bookkeeping.ipynb?

If it works, I will update the readme as well

### CODE REVIEW

File: src/lace/archive/nyx_archive.py [hunk 1]
```diff
@@ -21,13 +21,18 @@ class NyxArchive(BaseArchive):
 
     def __init__(
         self,
+        nyx_version,
```
Comment (andreufont) [inline src/lace/archive/nyx_archive.py:24]: Could you add a default value for that, or allow passing None?

**Comment by andreufont** [comment]
Could you take care of this

**Comment by andreufont** [comment]
Sorry, wrong button...

I noticed that this is a "draft" pull request?

**Comment by jchavesmontero** [comment]
Yes, I made a draft because I wanted to check that it worked. I am assuming it does, so before any push, I wanted to update the readme. Let me do it and I will turn it into a real PR

**Comment by andreufont** [comment]
@lauracabayol - could you take care of the review? Otherwise I'll do it on Monday

**Comment by jchavesmontero** [comment]
I have already updated the readme, waiting for your review

**Comment by andreufont** [comment]
@lauracabayol this one

**Review by lauracabayol — APPROVED** [review]

**Comment by jchavesmontero** [comment]
Thanks a lot @lauracabayol!

**Pull request merged**